src/components/Header.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+	<div class="header">
+		<v-app-bar dense flat color="#22d4ad">
+			<v-toolbar-title>Beans Love Beers</v-toolbar-title>
+
+			<div class="selectContainer">
+				<v-select
+					v-model="lang"
+					class="langSelector"
+					:items="langs"
+					:value="langs[0]"
+				></v-select>
+			</div>
+			<v-tabs v-model="activeTab">
+				<v-tab v-for="tab in tabs" :key="tab.name" :to="{ name: tab.name }">
+					{{ $t(`menu.${tab.localName}`) }}
+				</v-tab>
+			</v-tabs>
+		</v-app-bar>
+	</div>
+</template>
+
+<script>
+export default {
+	data: () => ({
+		activeTab: "",
+		tabs: [
+			{ name: "Home", localName: "home" },
+			{ name: "Favourites", localName: "favourites" },
```

Comment:
can you move such logic to computed something like this:
`tabs() {
  return [
    {
      name: "Home",
      value: this.$t('menu. favourites')
    },
    ...
  ]
}`